unlock-protocol-com/src/components/pages/Grants/index.tsx

```diff
@@ -0,0 +1,358 @@
+import { Button } from '@unlock-protocol/ui'
+import { Link } from '../../helpers/Link'
+import { BsLaptop as LaptopIcon } from 'react-icons/bs'
+import { HiSpeakerphone as SpeakerPhoneIcon } from 'react-icons/hi'
+import { FaHandsHelping as HelpingIcon } from 'react-icons/fa'
+import { MdGeneratingTokens as TokenIcon } from 'react-icons/md'
+import { AiOutlineLink as LinkIcon } from 'react-icons/ai'
+import { UnlockIcon } from '../../icons/Unlock'
+
+const UNLOCK_GRANT_APPLICATION_LINK =
+  'https://share.hsforms.com/1gAdLgNOESNCWJ9bJxCUAMwbvg22'
+
+const UNLOCK_GRANT_PROGRAM_LINK =
+  'https://docs.unlock-protocol.com/unlock/governance/grants-bounties-and-matchings'
+
+const UNLOCK_GRANT_HELP = [
+  {
+    Icon: TokenIcon,
+    title: 'Get UDT token grant',
+    text: 'We will provide you a grant of 100 UDT tokens to kickstart your project on Unlock',
+  },
+  {
+    Icon: LaptopIcon,
+    title: 'Technical Guidance for Project Development',
+    text: 'Get help from our developers on discord. We will provide suggestions and prioritize your issues.',
+  },
+  {
+    Icon: SpeakerPhoneIcon,
+    title: 'Marketing & Promotional Support',
+    text: 'We will help you perfect your pitch, provide feedback, and promote you on our social handles with over thousands of followers each.',
+  },
+  {
+    Icon: HelpingIcon,
+    title: 'Connecting with community and investors.',
+    text: 'We will help you launch, connect with our community members so you can find early users for your product.',
+  },
+]
+
+const UNLOCK_GRANT_COMMITTEE = [
+  {
+    name: 'Amber Case',
+    text: 'Advisor Unlock Protocol',
+    avatarURL: '/images/marketing/grants/amber-case.png',
+  },
+  {
+    name: 'Denise Xifara',
+    text: 'Advisor GMG Ventures',
+    avatarURL: '/images/marketing/grants/denise-xifara.png',
+  },
+  {
+    name: 'Henry Hoffman',
+    text: 'BAFTA-winning Game Designer',
+    avatarURL: '/images/marketing/grants/henry-hoffman.png',
+  },
+  {
+    name: 'Patrick Mayr',
+    text: 'Advisor Cherry Ventures',
+    avatarURL: '/images/marketing/grants/patrick-mayr.png',
+  },
+]
+
+const UNLOCK_GRANT_CRITERIA = [
+  {
+    title: 'Impact',
+    text: 'Can the project benefit to a large population of users?',
+  },
+  {
+    title: 'Feasibility',
+    text: 'Is the solution easy to use and a reasonable implementation?',
+  },
+  {
+    title: 'Team',
+    text: 'Does the team have the skills to deliver?',
+  },
+  {
+    title: 'Maturity',
+    text: 'Is this a new startup or a launched system with revenue?',
+  },
+]
+
+const UNLOCK_PAST_COHORTS = [
+  {
+    name: 'SwordyBot',
+    avatarURL: '/images/marketing/grants/patrick-gallagher.png',
+    by: 'Patrick Gallagher',
+    text: 'SwordyBot uses Unlock Protocol to let Discord administators automatically grant or limit access to Discord channels based on NFTs.',
```

Comment:
Let's add the links to each project's website please.

Reply:
Could you provide links I should add?

I don't have links for some of the entries (it wasn't in the figma either).